shared/protos/garbage/garbage.proto

```diff
@@ -0,0 +1,234 @@
+syntax = "proto3";
+
+package google.garbage.v1;
```

Comment:
Let's use a namespace that doesn't include `google` and doesn't end in a version.

Reply:
Trashed it up.